pygpu/gpuarray.pyx

```diff
@@ -40,6 +40,15 @@ cdef bytes _s(s):
         return s
     raise TypeError("Expected a string")
 
+cdef size_t countis(l, object val):
```

Comment:
Is it useful to test numpy version and use the numpy count for old numpy to don't slow down setitem and getitem? Those 2 methods was optimized a few times, it would not be great that this new implementation slow them down.